core/drivers/crypto/se050/glue/i2c.c

```diff
@@ -50,9 +50,14 @@ int glue_i2c_init(void)
 	if (transfer == rpc_io_i2c_transfer)
 		return 0;
 
+#if !CFG_CORE_SE05X_I2C_TRAMPOLINE
 	transfer = native_i2c_transfer;
 
 	return native_i2c_init();
+#else
+	/* not reached */
+	return 0;
+#endif
```

Comment:
```suggestion
	if (IS_ENABLED(CFG_CORE_SE05X_I2C_TRAMPOLINE)
		return 0;

	transfer = native_i2c_transfer;

	return native_i2c_init();
```

Reply:
Hmm. I think this will generate linker errors though, since only imx/stm32 has a native i2c implementation and has `native_i2c_init()` defined.

Reply:
Then perhaps `native_i2c_transfer()` and `native_i2c_init()` should to be stubbed in `core/drivers/crypto/se050/glue/include/i2c_native.h` when `CFG_CORE_SE05X_I2C_TRAMPOLINE` is enabled? @ldts

Reply:
sorry I am seeing all of the comments now. The issue is that usage of this configuration is being misinterpreted.  The configuration does what is supposed to do -and the implementation is correct.

What @hoihochan is after is a new configuration flag to postpone the creation of the session until the REE is ready to handle the I2C transfers. At that point, the session can be created using the PTA to configure the secure channel.